exonum-java-binding/core/rust/src/runtime/java_service_runtime.rs

```diff
@@ -49,27 +42,14 @@ pub struct JavaServiceRuntime {
 }
 
 impl JavaServiceRuntime {
-    /// Creates new runtime from provided config or returns the one created earlier.
-    ///
-    /// There can be only one `JavaServiceRuntime` instance at a time.
```

Comment:
Why is this statement removed? Multiple JVMs are not supported in the same process.

Reply:
Perhaps I got it a little wrong - I thought it was related to the old code of singleton creation, thus moved it to the JavaServiceFactoryAdapter.get_or_create_java_service_runtime(). Copied it back.